hphp/test/tools/import_zend_test.py

```diff
@@ -81,6 +81,9 @@
     # Zend's "run-tests" uses the cgi-fcgi sapi to run tests
     # our implementation is simply different by design
     '/ext/standard/versioning/php_sapi_name_variation001.phpt',
+
+    # bad assumption about system call effect
+    'zend/good/ext/standard/tests/general_functions/proc_nice_basic.php',
```

Comment:
I feel like this would be better off as a flaky test.  In any case the actual file location should be changed or deleted as well (depending on whether the test is marked `flaky`, or `no_import`).

Also I believe the path here is in the PHP repo.  So in this case:
`/ext/standard/tests/general_functions/proc_nice_basic.phpt`  (note the `t` at the end)